sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/__init__.py

```diff
@@ -208,6 +213,7 @@ def single_recognize_pii_entities(
     :type credential: ~azure.ai.textanalytics.TextAnalyticsApiKeyCredential or
         ~azure.core.credentials.TokenCredential
     :param str input_text: The single string to recognize entities from.
+        Limit text length to 5120 chars.
```

Comment:
I wonder if this will ever change on the service side... if it does we are going to be inconsistent.
Should we remove it and let the service handle that?

Reply:
Other type of things that are conditioned: https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview#data-limits
Maybe just the link will be enough

Reply:
I see what you're saying, I'll try the link and see how it formats in the docs